plugins/GSdx/res/tfx.fx

```diff
@@ -1002,13 +1003,13 @@ PS_OUTPUT ps_main(PS_INPUT input)
 
 	output.c1 = c.a * 2; // used for alpha blending
 
-	if(PS_AOUT) // 16 bit output
+	if((PS_DFMT == FMT_16) || PS_AOUT) // 16 bit output
 	{
 		float a = 128.0f / 255; // alpha output will be 0x80
 
 		c.a = PS_FBA ? a : step(0.5, c.a) * a;
 	}
-	else if(PS_FBA)
+	else if((PS_DFMT == FMT_32) && PS_FBA)
```

Comment:
I wasn't sure about this one but I added it just in case following how gl code works.